### ISSUE TITLE
delete job error - TypeError: Cannot read property 'success' of undefined

### ISSUE THREAD

**Issue by changjunhao** [stonecircle/kue-ui#39]
kue-ui-client v1.0.2

<img width="1007" alt="2017-02-23 10 05 29" src="https://cloud.githubusercontent.com/assets/7688639/23262257/08015766-fa15-11e6-97db-436cbc1ad842.png">

**Comment by changjunhao** [comment]
@Chris Manson   The error appears in this code
```
define("kue-ui-client/jobs/state/controller", ["exports", "ember"], function(e, t) {
    e.default = t.default.Controller.extend({
        queryParams: ["page", "order"],
        page: 1,
        order: "asc",
        jobs: t.default.inject.service(),
        actions: {
            removeJob: function(e) {
                var t = this;
                this.get("jobs").remove(e).then(function() {
                    t.get("notifications").success("Job Deleted", {
                        autoClear: !0
                    }),
                    t.get("model").removeObject(e)
                })
            },
            updateOrder: function() {
                var e = this.get("order");
                this.set("order", "asc" === e ? "desc" : "asc")
            }
        }
    })
}),
```

**Comment by mansona** [comment]
Thanks for the  fix in #40 👍  I've just released v1.0.3 

If you are using kue-ui-express (which I think you are), because the dependency on kue-ui-client is not fixed npm install should pick up the new version automatically https://github.com/stonecircle/kue-ui-express/blob/master/package.json#L14

I don't know the exact method to force it to do that but you can delete your node_modules and it will pick it up for sure after a npm install 👍
